README.md

```diff
@@ -756,6 +761,10 @@ Set a filter which will tell the API to return a user matching the provided iden
 
 Send the API request using the pre-set filters.
 
+- `located(details: LocatableServiceParameters)`
```

Comment:
This doesn't seem like the correct type. I believe this should be `LocatableUserParameters` correct?